Description: remove fullscreen by default
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/nwep.c
+++ b/nwep.c
@@ -446,7 +446,7 @@
 #define FULLSCREEN SDL_FULLSCREEN
 	SDL_ShowCursor(0);
 #endif
-	SDL_SetVideoMode(XRES, YRES, 32, SDL_OPENGL | FULLSCREEN);
+	SDL_SetVideoMode(XRES, YRES, 32, SDL_OPENGL);
 	glViewport(0, 0, XRES, YRES);
 	introInit();
 
